frontera/settings/default_settings.py

```diff
@@ -50,6 +51,7 @@
 }
 SQLALCHEMYBACKEND_REVISIT_INTERVAL = timedelta(days=1)
 STATE_CACHE_SIZE = 1000000
+STATE_CACHE_SIZE_LIMIT = 0
```

Comment:
The reasoning for this is that if we access `STATE_CACHE_SIZE_LIMIT` without specifying a value. We get None. Comparing integers with None works in PY2, but is an error in PY3. So https://github.com/scrapinghub/frontera/blob/master/frontera/contrib/backends/memory/__init__.py#L141 would fail.